src/components/tools/image-tool.tsx

```diff
@@ -181,11 +190,28 @@ export default class ImageToolComponent extends BaseComponent<IProps, IState> {
             onUrlChange={this.handleUrlChange}
           />
         </div>
-        <EmptyImagePrompt show={showEmptyImagePrompt} />
+          <EmptyImagePrompt show={showEmptyImagePrompt} />
```

Comment:
```suggestion
        <EmptyImagePrompt show={showEmptyImagePrompt} />
```
Tabbing was correct before.